src/plugins/auto_batch/auto_batch.cpp

```diff
@@ -736,22 +738,23 @@ InferenceEngine::IExecutableNetworkInternal::Ptr AutoBatchInferencePlugin::LoadN
         // let's query the optimal batch size
         std::map<std::string, InferenceEngine::Parameter> options;
         options["MODEL_PTR"] = std::const_pointer_cast<ngraph::Function>(network.getFunction());
-        auto optBatchSize =
-            GetCore()->GetMetric(deviceName, METRIC_KEY(OPTIMAL_BATCH_SIZE), options).as<unsigned int>();
-        auto res = GetCore()->GetConfig(deviceName, CONFIG_KEY(PERFORMANCE_HINT_NUM_REQUESTS)).as<std::string>();
-        requests = PerfHintsConfig::CheckPerformanceHintRequestValue(res);
-        const auto& reqs = config.find(CONFIG_KEY(PERFORMANCE_HINT_NUM_REQUESTS));
+        auto optBatchSize = GetCore()->GetMetric(deviceName, ov::optimal_batch_size.name(), options).as<unsigned int>();
+        auto num_requests = GetCore()->GetConfig(deviceName, ov::hint::num_requests.name());
+        auto num_requests_str = num_requests.is<std::string>() ? num_requests.as<std::string>()
```

Comment:
@apankratovantonp  @ilya-lavrenov this code look as a combination of the 1.0 API (GetMetric) and 2.0 config names.
pretty funny, but the  '? :' is quite inconvenient